imap_processing/cli.py

```diff
@@ -404,7 +422,19 @@ class Glows(ProcessInstrument):
     """Process GLOWS."""
 
     def do_processing(self, dependencies):
-        """Perform GLOWS specific processing."""
+        """
+        Perform GLOWS specific processing.
+
+        Parameters
+        ----------
+        dependencies : array_like
+            List of dependencies to process.
+
+        Returns
+        -------
+        iterable TODO double check
+            Cdf file containing process data. TODO double Check.
```

Comment:
I think all of these are going to be returning the same thing, which is a list of paths to the processed data.
```suggestion
        list[pathlib.Path]
            Cdf files containing processed data.
```

Reply:
Yes, these should all match the `do_processing` docs from the ProcessInstruments class. `dependencies` should be a list of paths, and it should return a list of paths.